bigtable/google/cloud/bigtable/table.py

```diff
@@ -354,7 +362,9 @@ def read_row(self, row_key, filter_=None):
         """
         row_set = RowSet()
         row_set.add_row_key(row_key)
-        result_iter = iter(self.read_rows(filter_=filter_, row_set=row_set))
+        result_iter = iter(self.read_rows(filter_=filter_,
+                                          row_set=row_set,
+                                          retry=retry))
```

Comment:
This formatting is going to make `nox -e lint` unhappy.  Please run `nox -e blacken` and push the result.

Reply:
Thanks for the hint. Done!